user-docs/latest/content-manager/managing-relational-fields.md

```diff
@@ -0,0 +1,56 @@
+# Managing relational fields
+
+Relation-type fields added to a content-type from the Content-Types Builder allow to establish a relation with another content-type. These fields are called "relational fields".
+
+Relational fields are before all else regular fields, meaning that their content is written from the edit view of the content-type they belong to (see [Writing content](writing-content.md)). 
+
+However, relational fields can point to one or several entries of the other content-type, this is why in the Content Manager it is possible to manage a content-type's relational fields to choose which entries are relevant.
+
+::: details Example
+In my Strapi admin panel I have created 2 collection types:
+- Restaurant, where each entry is a restaurant
+- Category, where each entry is a type of restaurant
+
+I want to assign a category to each of my restaurants, therefore I have established a relation between my 2 collection types: restaurants can have one category.
+
+In the Content Manager, from the edit view of my Restaurant entries, I can manage the Category relational field, and choose which entry of Category is relevant for my restaurant.
+:::
+
+![Relational fields box in the edit view](../assets/content-manager/edit-view_relational-fields.png)
+
+The relational fields of a content-type are managed from the Relational fields box, located in the right side of the edit view interface. It lists the names of the relational fields. Under each relational field name is displayed a listbox containing all available entry titles.
+
+The Relational fields box allows to choose which entries the relational fields should point to. You can either choose one or several entries depending on the type of relation that was established.
+
+## Managing one-choice relational fields
+
+Many-to-one and one-to-one types of relation only allow to choose one entry per relational field.
+
+<img src="../assets/content-manager/RF_one-choice.png" width="45%">
+
+To select the only relevant relational field's entry:
```

Comment:
This comment is for the One and Many sections.
It's also possible to type in the input box, it will do a search.
Based on the display field:
if you display the "name" of a restaurant in the relation, you will not be able to seach by the id of the restaurant.
I think it's worthit to add this info for both relation types (one and many)